### ISSUE TITLE
[3.0.0-rc0][java] Nested objects are not typified as before

### ISSUE THREAD

**Issue by marcoreni** [swagger-api/swagger-codegen#7523]
##### Description

In swagger 2 codegen nested objects with properties were correctly generated (there was a specific type declaration so that child properties could be correctly accessed).

In 3.0.0-rc0 (basic Java generation) these properties are lost: the object is considered a plain "Object" and there is no way to access the childs direcly.

##### Swagger-codegen version

3.0.0-rc0

##### Swagger declaration file content or url

https://gist.github.com/marcoreni/4a689e742a3bdd8b28eed9d1b1dddcd1

##### Command line used for generation

```shell
$ java -jar ./swagger-codegen-cli.jar generate -i ./test.yml -l java
```

**Comment by HugoMario** [comment]
@marcoreni when you have a chance, can you please check if this issue is still present?

**Comment by marcoreni** [comment]
@HugoMario This seems to be solved using latest commits in 3.0.0 branch. Is there a way to access a snapshot version so that we can test it using mvn (or is there a new RC release planned soon)? In Sonatype snapshots for 3.0.0 are stuck at 20180112 : https://oss.sonatype.org/content/repositories/snapshots/io/swagger/swagger-codegen/3.0.0-SNAPSHOT/

**Comment by HugoMario** [comment]
@marcoreni sorry for delay response. I'll close this since the issue is solved. About next release i still don't have a date but i'll let you know once i know.
